CONTRIBUTING.md

```diff
@@ -412,13 +412,13 @@ Note that the code shown in Chrome DevTools is compiled code and therefore diffe
 
 > Example: https://github.com/babel/babel/pull/11640
 
-- Create a new PR that describes the proposed AST shape in [ESTree](https://github.com/estree/estree) (ex: [Decimal AST](https://github.com/estree/estree/pull/220)). The new AST should follow ESTree's [design philosophy](https://github.com/estree/estree#philosophy).
+- Create a new PR that describes the proposed AST shape in [ESTree](https://github.com/estree/estree) (ex: [Import Attributes AST](https://github.com/estree/estree/pull/302)). The new AST should follow ESTree's [design philosophy](https://github.com/estree/estree#philosophy).
 - After the ESTree PR is accepted, update [ast/spec.md](https://github.com/babel/babel/blob/master/packages/babel-parser/ast/spec.md). Note that there are differences between Babel AST and ESTree. In these cases, consistency with current Babel AST outweighs alignment to ESTree. Otherwise it should follow ESTree.
 
-- [ ] Implement parser plugins based on the new AST. The parser plugin name should be the unprefixed slug of the TC39 proposal URL in _camelcase_, i.e. `exportDefaultFrom` from `https://github.com/tc39/proposal-export-default-from`.
+- [ ] Implement parser plugins based on the new AST. The parser plugin name should be the unprefixed slug of the TC39 proposal URL in _camelcase_, i.e. `importAttributes` from `https://github.com/tc39/proposal-import-attributes`.
```

Comment:
Let's maybe use an actual proposal, since we are also removing the `importAttributes` plugin (it's enabled by default)?

Reply:
What about `pipelineOperator`? That proposal is too controversial to advance any time soon, and too popular to be withdrawn.